python/python/ert_gui/simulation/ensemble_experiment_panel.py

```diff
@@ -46,9 +46,6 @@ def __init__(self):
     def isConfigurationValid(self):
         return self._active_realizations_field.isValid()
 
-    def toggleAdvancedOptions(self, show_advanced):
```

Comment:
How does it help to just remove this code? Should not these elements be displayed?

Reply:
toggleAdvancedOptions was used to hide elements when the panels were created.